src/routers/UserRouter.ts

```diff
@@ -48,6 +48,17 @@ const logger = getLogger();
 const COUNTRY_LIST_CODES = COUNTRY_LIST.map((country) => country.value);
 const COUNTRY_LIST_SORTED = orderBy(COUNTRY_LIST, ['label'], ['asc']);
 
+const setUserData = (userInfo) => ({
```

Comment:
Can we use a different function name? For me it looks like a "getUserData" or a "parseUserInfo". @iacoshoria thoughts?